pyladies_avatars_rotation.csv

```diff
@@ -1,16 +1,42 @@
+Russia,Saint Petersburg,https://hugovk.github.io/python-logos/pyladies/img/pyladies_spb.png,http://spb.pyladies.com/,pyladies_spb
+Mexico,Mexico City,https://hugovk.github.io/python-logos/pyladies/img/MxPyladies.jpg,https://www.meetup.com/Mexico-City-Pyladies-Meetup/,MxPyladies
+USA,Seattle,https://hugovk.github.io/python-logos/pyladies/img/PyLadiesSEA.png,https://www.meetup.com/Seattle-PyLadies/,PyLadiesSEA
+USA,Silicon Valley,https://hugovk.github.io/python-logos/pyladies/img/siliconvalley.png,https://www.meetup.com/PyLadies-Silicon-Valley/
+Brazil,Belém,https://hugovk.github.io/python-logos/pyladies/img/PyLadiesBelém.png,https://www.facebook.com/pyladiesbelem
+Brazil,Blumenau,https://hugovk.github.io/python-logos/pyladies/img/pyladies-blumenau.png,https://www.facebook.com/groups/PyLadiesBlumenau/,pyladiesbnu
+Brazil,Brasília DF,https://hugovk.github.io/python-logos/pyladies/img/PyLadies Brasília DF.png,https://df.pyladies.com/,PyLadiesDF
+Brazil,Brazil,https://hugovk.github.io/python-logos/pyladies/img/PyLadiesBrasil.jpg,https://brasil.pyladies.com/,PyLadiesBrasil
+Brazil,Caxias do Sul,https://hugovk.github.io/python-logos/pyladies/img/PyladiesCaxias.jpg,https://pyladiescaxias.github.io/,PyladiesAracati
+Brazil,Florianópolis,https://hugovk.github.io/python-logos/pyladies/img/PyLadies Florianópolis.jpg,https://www.facebook.com/groups/PyLadiesFlorianopolis/,PyLadiesFloripa
+Brazil,Fortaleza,https://hugovk.github.io/python-logos/pyladies/img/PyLadies Fortaleza.png,https://www.facebook.com/PyLadiesFortaleza/,pyladiesfort
+Brazil,Goiânia,https://hugovk.github.io/python-logos/pyladies/img/pyladies-goiania.png,https://www.instagram.com/pyladies_goiania/
+Brazil,Maceió,https://hugovk.github.io/python-logos/pyladies/img/PyLadiesMaceio.png,https://www.facebook.com/PyLadiesMaceio/,pyladiesmaceio
+Brazil,Manaus,https://hugovk.github.io/python-logos/pyladies/img/pyladiesmanaus.png,https://www.facebook.com/pyladiesmanaus/,pyladiesmanaus
+Brazil,Natal,https://hugovk.github.io/python-logos/pyladies/img/pyladies_natal.svg,https://pyladiesnatal.github.io/,PyladiesNatal
+Brazil,Paraíba,https://hugovk.github.io/python-logos/pyladies/img/pyladiespb.png,https://www.facebook.com/pyladiespb,pyladiespb
+Brazil,Parnaíba,https://hugovk.github.io/python-logos/pyladies/img/pyLadiesPHB.png,https://pyladiesparnaiba.carrd.co/,pyladiesphb
+Brazil,Porto Alegre,https://hugovk.github.io/python-logos/pyladies/img/pyladies-porto-alegre.png,https://pyladiespoa.pythonanywhere.com/,Pyladiespoa
+Brazil,Recife,https://hugovk.github.io/python-logos/pyladies/img/pyladiesrecife.png,https://linktr.ee/pyladiesrecife,pyladies_recife
+Brazil,Ribeirão Preto,https://hugovk.github.io/python-logos/pyladies/img/Pyladies-RP.png,https://pyladies-rp.github.io/,pyladies_recife
+Brazil,Salvador,https://hugovk.github.io/python-logos/pyladies/img/pyladies-salvador.png,https://pyladiessalvador.github.io/,pyladiessa
```

Comment:
```suggestion

```

I noticed that the Salvador chapter is using the other logo for their twitter avatar. So let's remove this one and choose the next one.